spec/models/local_authority_spec.rb

```diff
@@ -76,5 +76,13 @@
         expect(@local_authority.link_last_checked).to be_nil
       end
     end
+
+    describe "calculate_count_of_broken_links" do
```

Comment:
We could add another link here with status 200 to show that the broken link count is correct in counting only links that are broken.